app/templates/community/index.hbs

```diff
@@ -47,7 +47,7 @@
   <section aria-labelledby="section-reporting-bugs">
     <div class="layout">
       <div class="lg:col-2 lg:start-5">
-        <img alt="Bug" class="img-content" src="/images/community/bug.png">
+        <img alt="Bug" class="img-content" src="/images/community/bug.png" role="presentation" loading="lazy">
```

Comment:
If it has role of presentation then the `alt` attribute should be empty. 

We need to make sure that the images really are presentational, meaning used in places with other text.

Reply:
```suggestion
        <img alt="" class="img-content" src="/images/community/bug.png" role="presentation" loading="lazy">
```